modules/st2-highlight/highlight.directive.js

```diff
@@ -44,6 +44,8 @@ module.exports =
       scope.$watch('code', function (code) {
 
         scope.hidden = true;
+	scope.wrap = false;
```

Comment:
Please make sure you use spaces and not tabs for indentation. Not sure why linter didn't point you to that, will check.